### PR TITLE
Reduce shipit-pulse-listener notification noise

### LINKED ISSUE #744: shipit_pulse_listener: Reduce email noise
We are receiving too many emails with no actionable information (that is, they only contain successful tasks).
I think we should 1) increase the period between emails; 2) only notify about tasks not completed successfully.

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #744.

### CODE REVIEW

**Comment by garbas** [comment]
@La0 just pinging since this is open since December. It probably slipped your radar.

**Comment by La0** [comment]
@garbas I was not comfortable with the idea of losing some monitoring (on successful nb of tasks) on pulse listener. 
But the project is quite stable now, so i figure it's time to merge it

File: src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py [hunk 1]
```diff
@@ -114,30 +113,35 @@ def send_report(self):
...
+                    100.0 * nb_tasks / total,
+                    nb_tasks,
+                    total,
+                )
+                content += '\n'.join([
+                    TASK_MD.format(task)
+                    for task in tasks
+                ])
+
+        if content:
```
Comment (La0) [inline src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py]: `content` could be unitiliazed here and raise an exception

File: src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py [hunk 2]
```diff
@@ -114,30 +113,35 @@ def send_report(self):
...
-        for email in self.emails:
-            self.notify.email({
-                'address': email,
-                'subject': 'Pulse listener tasks',
-                'content': content,
-                'template': 'fullscreen',
-            })
+        for hook_id, tasks_per_status in self.stats.items():
+            total = sum([len(tasks) for tasks in tasks_per_status.values()])
+            if len(tasks_per_status['completed']) == total:
```
Comment (La0) [inline src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py]: Maybe use an expression with `all()` here to increase readiblity

Comment (marco-c) [reply src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py]: I could only come up with this:
```Python
if all(len(tasks) == 0 for status, tasks in tasks_per_status.items() if status != 'completed'):
    continue
```

Which one do you prefer?

Comment (La0) [reply src/shipit_pulse_listener/shipit_pulse_listener/monitoring.py]: Right, the initial one is better.

**Pull request merged**